packages/react-dom/src/events/ReactDOMEventReplaying.js

```diff
@@ -290,6 +300,13 @@ function accumulateOrCreateContinuousQueuedReplayableEvent(
   return existingQueuedEvent;
 }
 
+export function isCapturePhaseSynchronouslyHydratableEvent(
+  eventName: DOMEventName,
+) {
+  // TODO: maybe include more events
+  return isReplayableDiscreteEvent(eventName);
```

Comment:
You can really just use this same function and rename it. The name isn't really meant to communicate that it has to be "replayable". It just was the name of the feature like the file "Event Replaying" - like is this event playing in the "Event Replaying" feature but now that feature isn't really about replaying so much as it's about sync hydrating.

Reply:
#22502